sqrl-base/src/main/java/com/datasqrl/config/PackageJson.java

```diff
@@ -125,4 +127,29 @@ interface DataDiscoveryConfig {
 
     ErrorCollector getErrors();
   }
+
+  enum LogMethod {
```

Comment:
This does not look correct from a product perspective: we are hard-coding what sinks we allow for the logging here. That should be resolved dynamically from the list of engines that are configured and the list of systembuildinconnectors. We don't want to duplicate that information imho.

Reply:
More specifically: when we add a new log engine (e.g. postgres-cdc) we should not have to go in here and add another value. That breaks modularity.

Reply:
I understand your point. However, I also see the benefit in enumerating the available configuration options. This way, if you need to quickly determine the available options, you can simply refer to one class file. If it becomes dynamic, the code maintainer would need to delve deeper into the code just to answer this question. Nonetheless, I see the value in both approaches. I've made the changes as you requested. Please let me know if it's satisfactory!

Reply:
thanks for changing that.